Unified/closor.py

```diff
@@ -322,6 +324,26 @@ def rank( wfn ):
             to = ['hn-cms-relval@cern.ch']
             sendEmail(subject, text, destination=to )
             ## just announced ; take it out now.
+            #BI.pop( bname )
+            subject = "Low Statistics for Relval"
```

Comment:
Please add the batch name in the subject line, like `"Low Statistics for %s" %(bname)`

Reply:
ok sure